satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -1,5 +1,18 @@
 // dbx.v1 golang satellitedb.dbx .
 
+//--- Value Attribution ---//
+model value_attribution (
+	key bucket_id
+
+	field bucket_id		blob
+	field user_id		blob
```

Comment:
which user?

Reply:
I thought each bucket had one user_id associated with it

Reply:
nope, buckets are associated to projects

Reply:
but projects have users. /shrug

Reply:
but a bucket does not have a single user

Reply:
yeah i guess it would need to be the project owner if such a thing exists

Reply:
but also Im curious because this data model is described in the design doc and was approved ref: https://github.com/storj/storj/pull/1996

so im not sure what to make of this.  I guess we should update the design doc? Or how does that work?

Reply:
shoot, didn't catch that in the design doc but we should probably update it. we currently don't have any concept of user roles. everyone's an owner right now